MainExp-analyses-Native.Rmd

```diff
@@ -275,6 +344,14 @@ cat9 = glmer(nativeAccuracy ~ group +
 
 summary(cat9)
 
+Anova(cat9)
+
+catNine_EB <- subset(allData, category == 9 & group == 'EB')
+binom.test(sum(catNine_EB$grade), nrow(catNine_EB), p = .5)
+
+catNine_SC <- subset(allData, category == 9 & group == 'SC')
+binom.test(sum(catNine_SC$grade), nrow(catNine_SC), p = .5)
+
 ```
 
 ### Native Accuracy Opposite Words
```

Comment:
This script is quite long so I would consider moving these within category comparison in another script